code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/PropositionItem.java

```diff
@@ -148,6 +148,8 @@ public void track(
             final String interaction,
             @NonNull final MessagingEdgeEventType eventType,
             final List<String> tokens) {
+        // record the event in event history
```

Comment:
The logic to record in event history is missing here

Reply:
Also missing in `public void track(@NonNull final MessagingEdgeEventType eventType)`